docs/designers-developers/developers/tutorials/block-tutorial/applying-styles-with-stylesheets.md

```diff
@@ -1,48 +1,56 @@
 # Applying Styles From a Stylesheet
 
-In the previous section, the block had applied its own styles by an inline `style` attribute. While this might be adequate for very simple components, you will quickly find that it becomes easier to write your styles by extracting them to a separate stylesheet file.
+In the previous step, the block had applied its own styles by an inline `style` attribute. While this might be adequate for very simple components, you will quickly find that it becomes easier to write your styles by extracting them to a separate stylesheet file.
 
-The editor will automatically generate a class name for each block type to simplify styling. It can be accessed from the object argument passed to the edit and save functions:
+The editor will automatically generate a class name for each block type to simplify styling. It can be accessed from the object argument passed to the edit and save functions. In step 2, we will create a stylesheet to use that class name.
 
 {% codetabs %}
 {% ES5 %}
 ```js
 var el = wp.element.createElement,
```

Comment:
For consistency with the starting example, we should wrap this within an IIFE passing the blocks and element variables.

Reply:
:+1: